src/main/plugin-metadata/plugin-security.policy

```diff
@@ -0,0 +1,7 @@
+grant {
```

Comment:
Note to current and future reviewers: we need to take a decision on whether we are going to keep the security permissions or not. The reason they appear is that they were needed by the tests and therefore made the assumption that would be needed in a production environment as well. However removing them and trying the main manual testing flow as described at the readme of the repo works fine which brings us to the conclusion that probably these are needed by the tests only.

A possible solution would be to remove them and keep the security manager disabled when testing (as it is right now). Otherwise we could play safe and keep them, just in case something unpredictable comes up in production with the risk of:
1. Having a potential security vulnerability
2. Not prompting when the security permissions are asking for it and causing the installation to hang